index.html

```diff
@@ -14,6 +15,61 @@
 </head>
 
 <body>
-  <h1>My Resume</h1>
+</style>
+</head>
+
+
+
+<div id="header"></div>
+<div class="left"></div>
+<div class="stuff">
+  <br><br>
+  <img class="my-portrait" src="https://upload.wikimedia.org/wikipedia/en/e/e0/Gollum.PNG" style="width: 25%">
+  <h1>Sm&eacute;agol - Web Dev</h1>
+  <h2></h2>
+  <hr />
+  <br>
+
+  <p class="head">Introduction</p>
+  I need my Precious!
+  <br>
+  <br>
```

Comment:
I'd recommend using CSS margins to handle spacing here instead of `<br>` tags where at all possible.